### PR TITLE
solar cycle example

### PR DESCRIPTION
**Pull request by ehsteve**
figure_1

### CODE REVIEW

**Comment by wafels** [comment]
I often use the framealpha keyword5 in plt.legend to show data under the legend. Shows matplotlib off a bit too.

**Comment by wafels** [comment]
The 5 is a typo in my previous comment.

File: examples/solar_cycle_example.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+The Solar Cycle
+===============
+
+This example shows the current and possible next solar cycle.
+"""
+
+import matplotlib.pyplot as plt
+import sunpy.lightcurve as lc
+import datetime
+
```
Comment (wafels) [inline examples/solar_cycle_example.py]: Import order should have datetime as the first import.

File: examples/solar_cycle_example.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+plt.plot(noaa_predict.data.index,noaa_predict.data['sunspot'],color='grey', label='Near-term Prediction')
+plt.fill_between(noaa_predict.data.index, noaa_predict.data['sunspot low'], noaa_predict.data['sunspot high'],
+                 alpha = 0.3, color='grey')
+
+plt.fill_between(noaa2.data.index, noaa2.data['sunspot RI smooth']*0.4, noaa2.data['sunspot RI smooth']*1.3,
+                 alpha = 0.3, color='grey', label='Next Cycle Predict')
+plt.ylim(0)
+plt.text('2011-01-01', 120,'Cycle 24',fontsize=16)
+plt.text('2024-01-01', 120,'Cycle 25',fontsize=16)
+plt.ylabel('Sunspot Number')
```
Comment (wafels) [inline examples/solar_cycle_example.py]: Add plt.xlabel ('year')

File: examples/solar_cycle_example.py (hunk 1 above)
Comment (Cadair) [inline examples/solar_cycle_example.py]: also line breaks between the different sets of imports if we are being *really* picky

**Comment by Cadair** [comment]
very nice example!

**Comment by ehsteve** [comment]
Rebased to one commit

**Comment by Cadair** [comment]
:+1:

**Comment by wafels** [comment]
+1

**Comment by Cadair** [comment]
backported to the 0.7 branch.

**Pull request merged**